scripts/launch-dataproc.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
Can you add a license header here? Just like other scripts in the folder(e.g. https://github.com/intel-analytics/BigDL/blob/master/scripts/run.example.sh#L3)